src/components/color-picker/color-picker.tsx

```diff
@@ -0,0 +1,75 @@
+import { Component, h, State } from '@stencil/core';
+
+/**
+ * This component enables you to select a swatch from out color palette, simply
+ * by clicking on it. You can then copy the css variable name of the chosen color
+ * and use it where desired.
+ *
+ * The color picker can also show you a preview of any valid color name or color value.
+ *
+ * @exampleComponent limel-example-color-picker
+ */
+
+@Component({
+    tag: 'limel-color-picker',
+    shadow: true,
+    styleUrl: 'color-picker.scss',
+})
+export class ColorPicker {
+    /**
+     * Name or code of the chosen color
+     */
+    @State()
+    public color: string;
+
+    @State()
+    private isOpen = false;
+
+    public render() {
+        const background = this.color ? { '--background': this.color } : {};
+
+        return (
+            <div class="color-picker">
+                <limel-popover open={this.isOpen} onClose={this.onPopoverClose}>
+                    <div
+                        class="picker-trigger"
+                        slot="trigger"
+                        style={background}
+                        role="button"
+                        tabindex="0"
+                        onClick={this.openPopover}
+                        title="Click to pick a color"
```

Comment:
This should not be hard coded

Reply:
also asked above :) should it be a prop or a translation fetched from within Elements translation fiels?

Reply:
It depends. Some of them should probably be set on the consumer side and then it should be a prop. If it's an internal label that should always stay the same, then it should be in a translation file

Reply:
Tnx. But I assume you are not talking only about the `title` right? Since you marked a few lines… Or?